services/workflows-service/src/bull-mq/outgoing-webhook/outgoing-webhook-queue.service.ts

```
@@ -0,0 +1,80 @@
+    throw new Error('Method not implemented.');
+    const response = await this.webhookService.invokeWebhook({
+      ...job.data.jobData,
+    });
+
+    this.logger.log(`Webhook job ${job.id} completed with status: ${response.status}`);
+
+    if (response.status >= 200 && response.status < 300) {
+      return;
+    }
+
+    await this.handleRetryStrategy(response.status, job);
```

Biome

[error] This code is unreachable

... because this statement will throw an exception beforehand


(lint/correctness/noUnreachable)